akka-sample-sharding-java/LICENSE

```diff
@@ -0,0 +1,10 @@
+Akka sample by Lightbend
+
+Licensed under Public Domain (CC0)
+
+To the extent possible under law, the person who associated CC0 with
+this Activator Tempate has waived all copyright and related or neighboring
+rights to this Activator Template.
```

Comment:
we should not use Activator here, haven't we changed that in other samples?
perhaps do a full sweep in another PR?